luminance/src/state.rs

```diff
@@ -237,6 +242,12 @@ impl GraphicsState {
     }
   }
 
+  pub(crate) unsafe fn set_patch_vertex_nb(&mut self, state: usize) {
+    if self.patch_vertex_nb != state {
+      gl::PatchParameteri(gl::PATCH_VERTICES, state as i32);
```

Comment:
It also feels weird to pass around `usize` when it's going to be cast to a `GLint` when it's actually used anyway. But it would also also feel weird to accept an i32 or `GLint` from the user, since that implies negative values are valid. I'd be surprised if they were, but I can't find any info on whether negative values are ok here, and what that would mean. It certainly produces visually interesting results, though.
![2019-11-29-235803_1920x1200_scrot](https://user-images.githubusercontent.com/13943374/69892275-246ad700-1304-11ea-9ab3-38307eb1c9d3.png)